### PR TITLE
Detection models update

### PR DESCRIPTION
**Pull request by Ilya-Krylov**

### CODE REVIEW

**Comment by Ilya-Krylov** [comment]
@druzhkov-paul @AlexanderDokuchaev @snosov1 please review

**Comment by AlexanderDokuchaev** [comment]
run tests

File: pytorch_toolkit/object_detection/configs/face-detection-0100.py [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        anchor_strides=(16, 32),
+        anchor_widths=( [  9.4 * 0.85333333333,  25.1 * 0.85333333333,  14.7 * 0.85333333333,  34.7 * 0.85333333333],
+                        [143.0 * 0.85333333333,  77.4 * 0.85333333333, 128.8 * 0.85333333333,  51.1 * 0.85333333333, 75.6 * 0.85333333333]),
+        anchor_heights=([ 15.0 * 0.85333333333,  39.6 * 0.85333333333,  25.5 * 0.85333333333,  63.2 * 0.85333333333],
+                        [227.5 * 0.85333333333, 162.9 * 0.85333333333, 124.5 * 0.85333333333, 105.1 * 0.85333333333, 72.6 * 0.85333333333]),
+        target_means=(.0, .0, .0, .0),
+        target_stds=(0.1, 0.1, 0.2, 0.2),
+        depthwise_heads=True,
+        depthwise_heads_activations='relu',
+        loss_balancing=True))
```
Comment (sovrasov) [inline pytorch_toolkit/object_detection/configs/face-detection-0100.py]: Offtop: does it really work?

Comment (Ilya-Krylov) [reply pytorch_toolkit/object_detection/configs/face-detection-0100.py]: to be honest, I did not play with it. I just trusted you.

**Pull request merged**